doc/Customization-guide/How-to-contribute/Git-flow/index.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: page
+title: Git flow
+permalink: /:path/
+nav: /:path/Customization-guide/How-to-contribute/Git-flow
+sort_idx: 10
+---
+
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+We use a [GitHub flow](https://guides.github.com/introduction/flow/). That means, you should branch from the main repo and contribute back via pull requests. That's de facto standard workflow on GitHub—nothing fancy here.
+
+However, there is a thing you should know about Kaa branches. We have two main long-running branches: `master` and `develop`. `master` always points to the latest released version—**no pull requests are accepted to master**. `develop` is a main development branch; it reflects the latest development status. Usually, new changes should go there.
+
+There may be a number of short-living branches; most of them are used for specific jira tickets. However, `release-xxx` branches are special. They are pre-release branches which are used to test and stabilize everything before actual release. (Actual releases are tagged.) **Only bugfixes are accepted to release branches.**
+
+There may be develop branches for releases after next; they took a form of `develop-xxx` (e.g., `develop-1.0.0`). Usually, you should avoid committing to them and use branches for specific features.
+
+## Clonning
+
+The main Kaa repository is located here: <https://github.com/kaaproject/kaa>. The best way to contribute is to fork it (click fork button at the page), then clone your new repo:
+
+```sh
+git clone git@github.com:your_github_name/kaa.git
```

Comment:
The "your_github_name" part of the path may be not clear enough to users. Please add more descriptions and identify the part as something user should change before pasting.